clustering.py

```
@@ -94,127 +68,181 @@ def haversine_wrapper(point1, point2):
     return distance_matrix
 
 
-print(f"Using {args.method} distance calculation method")
-distance_matrix = calculate_distances(coords, method=args.method)
+def main() -> None:
```

Ruff (0.11.9)

Too many statements (64 > 50)

(PLR0915)